server/public/style.css

```diff
@@ -0,0 +1,4 @@
+.heading {
+  display: flex;
+  justify-content: space-around;
```

Comment:
Bootstrap has classes for this.

https://getbootstrap.com/docs/4.5/utilities/flex/